### PR TITLE
fix: update icon and documentation url

### PR DESCRIPTION
**Pull request by darahayes**
* Update image to this one 
producticons_0219_rgb_mobile data sync color

* Update name to 'Data Sync'

* Update 'documentationUrl'

### CODE REVIEW

File: apb.yml [hunk 1]
```diff
@@ -8,8 +8,8 @@ tags:
   - mobile-client-enabled
 metadata:
   displayName: Sync App
```
Comment (darahayes) [inline apb.yml]: @wei-lee is `Sync App` the agreed name? Should it be `Data Sync App`?

**Comment by darahayes** [comment]
@wei-lee Thanks for the heads up. Do I simply copy paste the `com.redhat.apb.spec` part or do I need other bits also?

**Comment by wei-lee** [comment]
> @wei-lee Thanks for the heads up. Do I simply copy paste the `com.redhat.apb.spec` part or do I need other bits also?

Just that part I think. @grdryn am I right?

**Comment by wei-lee** [comment]
👀

**Comment by wei-lee** [comment]
I deployed the APB to here: https://master.waterford-12c6.openshiftworkshop.com.

The new icon is showing up. However, when I try to run the apb, I am getting the following error:

```
[WARNING]: provided hosts list is empty, only localhost is available. Note
--
  | that the implicit localhost does not match 'all'
  | ERROR! no action detected in task. This often indicates a misspelled module name, or incorrect module path.
  |  
  | The error appears to have been in '/opt/ansible/roles/provision-sync-app-apb/tasks/main.yml': line 1, column 3, but may
  | be elsewhere in the file depending on the exact syntax problem.
  |  
  | The offending line appears to be:
  |  
  |  
  | - name: create configmap
  | ^ here
```

I have never seen this error before. Can someone else try it and see if it's working? @psturc

**Comment by psturc** [comment]
👀

**Comment by wei-lee** [comment]
@psturc that's strange. Do you have a link of the docker image you built?

**Comment by psturc** [comment]
@wei-lee tag `latest` or `v1.1.0` https://hub.docker.com/r/psturc/sync-app-apb/tags

**Comment by wei-lee** [comment]
@psturc you are right, it does work now with the image you have built. Looks like something to do with my local docker then

**Comment by darahayes** [comment]
I figure we could merge this then test the master version built by Dockerhub before creating the release tag @wei-lee how does that sound?

**Comment by wei-lee** [comment]
👍

**Comment by grdryn** [comment]
> > @wei-lee Thanks for the heads up. Do I simply copy paste the `com.redhat.apb.spec` part or do I need other bits also?
> 
> Just that part I think. @grdryn am I right?

@darahayes @wei-lee on the version of the `apb` tool that I have (2.0.2), I think it would be:

```
apb bundle prepare -c Dockerfile
apb bundle prepare -c Dockerfile.rhel7 
```

Probably the equivalent commands in your `apb` versions would have a `-c` option also, so that you don't have to manually copy over the label

**Comment by wei-lee** [comment]
@grdryn thanks. I updated this doc: https://github.com/aerogear/mobile-services-installer/blob/master/apb-release.md

**Pull request merged**